server/service/microsoft_mdm.go

```diff
@@ -1574,10 +1578,13 @@ func (svc *Service) storeSyncMLCmd(ctx context.Context, msg *mdm_types.SyncML) e
 				SystemOrigin: protoCMD.SystemOrigin,
 			}
 
-			err := svc.ds.MDMWindowsInsertCommand(ctx, newCmd)
-			if err != nil {
-				return fmt.Errorf("there was an issue inserting the command %v", err)
-			}
+			// TODO(mna): should now use MDMWindowsInsertPendingCommandForDevices (or a new one that
```

Comment:
That's now `MDMWindowsInsertCommandForHosts`, so I think a new batch one should be created to insert by device ID.